modules/balana-core/src/main/java/org/wso2/balana/PolicySet.java

```diff
@@ -392,21 +389,25 @@ public String encode() {
      * @param builder string stream into which the XML-encoded data is written
      */
     public void encode(StringBuilder builder) {
+        String xacmlVersionId = metaData.getXACMLIdentifier();
+
+        String version = getVersion();
 
-        builder.append("<PolicySet PolicySetId=\"").append(getId().toString()).
-                append("\" PolicyCombiningAlgId=\"").
-                append(getCombiningAlg().getIdentifier().toString()).append("\">\n");
+        builder.append("<PolicySet xmlns=\"" + xacmlVersionId + "\""  + " PolicySetId=\"" + getId() +
```

Comment:
Done. I also changed Policy.encode() to use append rather than String concatenation.